### PR TITLE
[READY] DOC - new version of documentation 

### PR DESCRIPTION
**Pull request by Badr-MOUFAD**
This mainly suggests new package documentation.

It addresses:

- [x] fix README
- [x] new docs architecture and content
- [x] new docs theme

[README preview](https://github.com/Badr-MOUFAD/celer/blob/new-docs/README.md)
[documentation preview](https://output.circle-artifacts.com/output/job/4ebc361e-df98-4de8-9c22-2d06aa6de399/artifacts/0/dev/index.html)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,22 +76,22 @@ For the last mean of contribution, here are the steps to help you setup ``celer`
 1. Fork the repository and afterwards run the following command to clone it on your local machine
 
 ```shell
-git clone https://github.com/{YOUR_GITHUB_USERNAME}/celer.git
+$ git clone https://github.com/{YOUR_GITHUB_USERNAME}/celer.git
```
Comment (mathurinm) [inline README.md:79]: this $ gets copied into the user input if ctrl c is used. for code block this is a negative point. So I'd rather not have it here

File: README.md [hunk 2]
```diff
@@ -76,22 +76,22 @@ For the last mean of contribution, here are the steps to help you setup ``celer`
...
 ```
 
 2. ``cd`` to ``celer`` directory and install it in edit mode by running
 
 ```shell
-cd celer
-pip install -e .
+$ cd celer
```
Comment (mathurinm) [inline README.md:85]: same

File: README.md [hunk 3]
```diff
@@ -76,22 +76,22 @@ For the last mean of contribution, here are the steps to help you setup ``celer`
...
+$ pip install -e .
 ```
 
 3. To run the gallery examples and build the documentation, run the followings
 
 ```shell
-cd doc
-pip install -r doc-requirements.txt
-make html
+$ cd doc
```
Comment (mathurinm) [inline README.md:92]: same

File: doc/about.rst [hunk 4]
```diff
@@ -0,0 +1,42 @@
+=====
```
Comment (mathurinm) [inline doc/about.rst:1]: Are you confident about the benefits of adding this file ? Each new file requires maintenance

**Comment by mathurinm** [comment]
Looks nice, did a first quick review

File: doc/about.rst (hunk 4 above)
Comment (Badr-MOUFAD) [reply doc/about.rst:1]: I have no strong opinion.

I will integrate this part into the home page.

File: doc/get_started.rst [hunk 5]
```diff
@@ -0,0 +1,98 @@
+===========
+Get started
```
Comment (mathurinm) [inline doc/get_started.rst:2]: very nice

File: doc/get_started.rst [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+    time elapsed for celer LassoCV: 5.062559127807617
+    time elapsed for scikit-learn LassoCV: 27.427260398864746
+
+
+
+Further links
+-------------
+
+This was just a starter examples. Get familiar with ``celer`` by browsing its :ref:`API documentation` or
```
Comment (mathurinm) [inline doc/get_started.rst:96]: typo : example

File: doc/conf.py [hunk 7]
```diff
@@ -145,6 +150,11 @@
 htmlhelp_basename = 'celer_doc'
 
 
+# -- Options for copybutton ---------------------------------------------
+copybutton_prompt_text = r">>> |\.\.\. |\$ |In \[\d*\]: | {2,5}\.\.\.: | {5,8}: "
```
[the remark this answers is not in the document]
Comment (Badr-MOUFAD) [reply doc/conf.py:156]: I provided a direct link to [Sphinx Copybutton](https://sphinx-copybutton.readthedocs.io/en/latest/index.html) extension and highlighted the corresponding part.

**Pull request merged**